jinja2cli/cli.py

```diff
@@ -411,7 +411,8 @@ def get_version(self):
 
 def main():
     parser = LazyOptionParser(
-        usage="usage: %prog [options] <input template> <input data>"
+        usage="usage: %prog [options] <input template> <input data>",
+        epilog="The input data can also be provided by stdin, e.g.: cat <input data> | jinja2 <input template>"
```

Comment:
I don't think we need an articulated example, we can just say the input argument is optional, and if not present reads from stdin. `-` can also explicitly be used for stdin as well.

Reply:
Hm, that's right. But I don't know how to formulate the message then. Do you have any idea?

Reply:
Maybe something like

```
Input data can also be omitted to be read from stdin
```